packages/react-ui/components/Tabs/Tab.tsx

```diff
@@ -299,7 +299,7 @@ export class Tab<T extends string = string> extends React.Component<TabProps<T>,
     // focus event fires before keyDown eventlistener
     // so we should check focusKeyPressed in async way
     requestAnimationFrame(() => {
-      if (tabListener.isTabPressed || this.isArrowKeyPressed) {
+      if (keyListener.isTabPressed || this.isArrowKeyPressed) {
```

Comment:
Получается, в табе тоже можно перейти на новый флаг и удалить его собственный listener! 

```suggestion
      if (keyListener.isTabPressed || keyListener.isArrowPressed) {
```

Reply:
да, здорово получилось! Спасибо, что заметил!